docs/source/upgrade_notes.rst

```diff
@@ -17,6 +17,10 @@ Upgrade Notes
   instead of ``None`` when the pack was not found in the index. This is technically a breaking
   change, but a necessary one because returning ``None`` caused the client to throw an exception.
 
+* The ``st2kv`` function has been changed (in 2.3.2) so that it no longer attempts to decrypt stored values by
+  default. To return decrypted values, this must be explictly enabled via parameter, like so:
+  ``st2kv('st2_key_id', decrypt=true)``.
```

Comment:
I personally don't mind either way. Only reason I was aiming for 2.3.2 is because @lakshmi-kannan recommended it. I'll let him chime in on the matter; there's no question that this is a breaking change.